doc/site/en/guides/border-router/beaglebone-black.md

```diff
@@ -1,338 +1,204 @@
-sectors are numbered starting with `0` and ending at Sector `N` somewhere around
-XX GB, the exact last number is dependent upon the actual SD card.
-
-Sector 0 always contains an MS-DOS Partition table. An MS-DOS partition table
-can hold between 1 and 4 partition entries. Each partition is a continuous
-series of sectors from `X` to `Y` somewhere within the bounds of the SD card.
-This repeats for each of the 4 possible partitions. Typically, partitions are
-located in order, with some number (`0` to `N`) of unused sectors at the end.
-This "some number of unused sectors" (`SOME_N`) can be used to your advantage
-later.
-
-Caution: The largest partition should be a Linux partition, and it must be the
-last partition on the SD card. This process does not work if the Linux partition
-is not the last partition.
-
-When writing an image to an SD card, writing begins at Sector `0` and progresses
-to `SOME_N`, depending on the size of the image. What you cannot do is stretch
-the partition around the image&mdash;that's not possible. Instead, think of a
-picture frame around a canvas. The picture frame is the partition and the
-picture is the data. What you can do is replace the existing picture frame with
-a larger one, and expand the canvas within:
-
-1.  Delete the existing Linux partition without deleting the data. You have
-    removed the picture frame, but the picture is still present on the canvas.
-1.  Create a new Linux partition that starts exactly where the old one started,
-    but ends close to the end of the SD card. You have created a larger picture
-    frame. The picture—the data—is still there on the canvas. It has not moved
-    and was not corrupted by this operation.
-1.  Use a file system-specific tool to grow the file system within the bounds of
-    the new partition. The canvas is stretched to fill the new, larger picture
-    frame.
-
-### 1. Identify the current data partition
-
-Boot the BBB from the SD Card and log in as `root`:
+*  External 5V AC adapter for power
+*  An 8 GB or larger microSD card ("uSD card" in this guide)
+*  A supported OpenThread platform (such as the [TI
+   CC2652](https://openthread.io/vendors/texas-instruments#cc2652)) for Thread
+   network connectivity in an RCP design
 
-```
-$ sudo bash
-```
+Steps to enable
+1. Download and install the OS.
+1. Prepare the Debian Environment for OTBR
+1. Build and install OTBR
+1. Set up a Wi-Fi access point
 
-Identify the SD card data partition. The `p1` suffix on the `Filesystem`
-field is the naming convention for Partition 1. The device itself is
-`/dev/mmcblk0`. In this example, only 295 MB are free. This is not enough
-space to build and install OTBR.
+Note: The BBB does not have built-in Wi-Fi support. This guide was built and
+tested with a BBONE-GATEWAY-CAPE for Wi-Fi AP operation. Some BeagleBone
+variants have onboard Wi-Fi capability, and some of this guide may be
+applicable.
```

Comment:
Callouts should be blockquote with > preceding the first line and the others indented

> Note: